android/src/main/java/com/fidelreactlibrary/adapters/FidelSetupKeys.java

```diff
@@ -53,6 +55,25 @@ public String jsName() {
         }
     }
 
+    public enum CardConfig {
+        ID("id"),
+        CONSENT_ID("consentID"),
```

Comment:
💅 To be consistent with the other properties, this should be "consentId", instead of "consentID"